music/demos/transcription.ts

```diff
@@ -15,13 +15,16 @@
  * limitations under the License.
  */
 import * as tf from '@tensorflow/tfjs-core';
+//@ts-ignore
```

Comment:
install types/mediarecorder and avoid all of thisany nonsense?

Reply:
ugh this bit of nonsense isn't cause of the types, it's cause the export is bad (it doesn't have a named export

but also i couldn't find types